lib/common/formats.js

```diff
@@ -52,6 +52,20 @@ function iconsWithPrefix(prefix, properties, config) {
     })
     .value().join('\n');
 }
+
+function minifyDictionary(obj) {
+  var toRet = {};
+  if (_.has(obj, 'value')) {
```

Comment:
Can this just be `obj.hasOwnProperty('value')` ? Just because we are using that below and trying to use lodash as little as possible in favor plain JS. :)

Reply:
Derp. I should have caught that in my earlier comment. Sorry @davixyz

Reply:
Oh dang! you are right it's better